ci/3.10.pylintrc

```diff
@@ -0,0 +1,40 @@
+[MESSAGES CONTROL]
+
+disable=C, # (C) convention, for programming standard violation
+    broad-except, # W0703: *Catching too general exception %s*
+    broad-exception-raised, # W0719: Raising too general exception: Exception
+    consider-using-dict-comprehension, # R1717: *Consider using a dictionary comprehension*
+    consider-using-from-import, # R0402: Use 'from foo import bar' instead
```

Comment:
new

Reply:
where do we get this data?

Reply:
what do you mean?

Reply:
I mean how do one get these pylint warnings list. Tomorow If I add for 3.11, where do I go and get these?

Reply:
I see. 

Run pylint on the code, check the documentation for what each warning means, then make a judgement call: fix the ones that *must* be fixed, for the rest fix as many as possible. When there are too many to fix, think how we address them. Do we want to suppress each one of them or suppress globally? If globally, add to this file.

In this specific case, this warning is just complaining about the syntax used to do the import. We have way too many of these and the functionality/correctness of the code is not affected. It'd be nice to fix them, but I didn't consider spending time on them was worth it so I suppressed globally.